cli/cmdlineparser.cpp

```diff
@@ -1084,6 +1093,9 @@ void CmdLineParser::printHelp()
         "                         be considered for evaluation.\n"
         "    --config-excludes-file=<file>\n"
         "                         A file that contains a list of config-excludes\n"
+        "--disable=<id>           Disable individual checks.\n"
```

Comment:
different spaces

Reply:
Yeah - I was afraid messed that up during a rebase...